src/Database/Orville/Internal/IndexDefinition.hs

```diff
@@ -5,28 +5,28 @@ License   : MIT
 -}
 
 module Database.Orville.Internal.IndexDefinition
-  ( uniqueIndex, simpleIndex
+  ( IndexName, uniqueIndex, simpleIndex, indexNameToString, stringToIndexName, safeStringToIndexName
```

Comment:
This formatting looks strange to me, did Brittany do this?

Reply:
Unfortunately Brittany is not set up yet on this project, I attempted to keep the formatting consistent with the rest of the file.